fastfield_codecs/src/lib.rs

```diff
@@ -388,6 +392,31 @@ mod bench {
         });
     }
 
+    #[bench]
+    fn bench_intfastfield_jumpy_fflookup_u128(b: &mut Bencher) {
```

Comment:
jumpy is probably not the most interesting one. Ideally something with a random stride with an average of 7 is nice.